pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -147,10 +147,10 @@ type ResourceBindingSpec struct {
 	// +optional
 	RescheduleTriggeredAt *metav1.Time `json:"rescheduleTriggeredAt,omitempty"`
 
-	// Suspension declares the policy for suspending different aspects of propagation.
+	// Suspension declares the policy for suspending different aspects of propagation and suspension of resource binding itself.
```

Comment:
```suggestion
	// Suspension declares the policy for suspending different aspects of propagation.
```
I don't think we should change it as scheduling is one of the aspects of propagation.

Reply:
Will keep it.